--- ./build/third_party_install/tensorflow/include/tensorflow_inc/tensorflow/stream_executor/platform/logging.h	2021-06-22 16:41:20.000000000 +0800
+++ logging.h	2021-08-16 19:41:43.082449275 +0800
@@ -19,7 +19,7 @@
 #include "tensorflow/core/platform/logging.h"
 #include "tensorflow/stream_executor/platform/port.h"
 
-#if !defined(PLATFORM_GOOGLE) && !defined(PLATFORM_GOOGLE_ANDROID)
+#if !defined(PLATFORM_GOOGLE) && !defined(PLATFORM_GOOGLE_ANDROID) && !defined(GOOGLE_LOGGING)
 
 #define PCHECK(invocation) CHECK(invocation)
 
